modules/renter/contractor/formcontract.go

```diff
@@ -363,6 +367,7 @@ func (c *Contractor) formContracts(a modules.Allowance) error {
 			// negotiate with because they'll probably be more expensive than
 			// we can afford.
 			c.log.Println("WARN: failed to negotiate contract:", h.NetAddress, err)
```

Comment:
Debugln, it's something that will probably happen a lot and is only interesting to developers.